### PR TITLE
Package/nodejs v18

### PR DESCRIPTION
**Pull request by KozhinovAlexander**
add nodejs v18

It is a follow-up to #16966

---

- [x] I've read the [contributing guidelines](https://github.com/conan-io/conan-center-index/blob/master/CONTRIBUTING.md).
- [x] I've used a [recent](https://github.com/conan-io/conan/releases/latest) Conan client version close to the [currently deployed](https://github.com/conan-io/conan-center-index/blob/master/.c3i/config_v1.yml#L6).
- [x] I've tried at least one configuration locally with the [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

**Comment by KozhinovAlexander** [comment]
> ### Conan v1 pipeline :x:
> 
> Changes not allowed in build 8:
> 
> ```
> [libudev, nodejs]
> ```
> 
> Only one library can be changed in the same PR.
> 
> ----------
> 
> ### Conan v2 pipeline (informative, not required for merge) :x:
> 
> > **Note**: Conan v2 builds are informative and they are not required for the PR to be merged.
> 
> The v2 pipeline failed. Please, review the errors and note this will be required for pull requests to be merged in the near future.
> 
> <details>
> 
> <summary> See details: </summary>
> 
> Changes not allowed in build 8:
> 
> ```
> [libudev, nodejs]
> ```
> 
> Only one library can be changed in the same PR.
> 
> </details>

The rule Here is basically stupid and Males people avoid doing PR.

**Comment by KozhinovAlexander** [comment]
@SSE4 @prince-chrismc Please review. It is a follow-up to #16966

**Comment by KozhinovAlexander** [comment]
@franramirez688 @uilianries please review

**Review by SSE4 — APPROVED** [review]

File: recipes/nodejs/config.yml [hunk 1]
```diff
@@ -5,3 +5,5 @@ versions:
     folder: all
   "16.3.0":
     folder: all
+  "18.15.0":
```
Comment (gjasny) [inline recipes/nodejs/config.yml:8]: Would it make sense to package `18.16.0` to reduce the amount of versions in cci?

**Comment by KozhinovAlexander** [comment]
> Could you please merge in `master` so that the diff-noise is reduced to the actual changes.

Done

**Comment by KozhinovAlexander** [comment]
@SSE4 I merged master into this branch. Please Approved again. Sorry for the noice.

**Comment by prince-chrismc** [comment]
Please do not force push 🙏 GitHub forces us to restart the review which is not fun!

**Review by prince-chrismc — APPROVED** [review]

**Comment by KozhinovAlexander** [comment]
@gjasny For sake of siplicity I will keep v18.15.0 and prefer to add new one in a separate PR.

**Review by gjasny — APPROVED** [review]

**Review by jwillikers — APPROVED** [review]

**Pull request merged**